### PR TITLE
Shyakadev/fix model setup & pine issue

### PR DESCRIPTION
**Pull request by shyakadev**
* update repo setup
* Removed unused import
* Update task.model.ts
* fix: rename model name
* Renamed the model name for a better clarification
* fix: formatting
* Added missing flag in tsconfig
* Added missing package for type checking
* Found some field on the default model of sequelize
        * createdAt & updatedAt
* Add no-null assertion operator & optional modifier on table field
* fix: pine parameter type check
* declared a pine module to work with no types
* included it, in tsconfig
* resolved path location of swagger and routes
* changed entry point from app.ts to index.ts

**It is running now without any error ✌️**

### CODE REVIEW

File: backend/src/logger/api.logger.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as pine from "pine";
+import pine from "pine";
```
Comment (marksikaundi) [inline backend/src/logger/api.logger.ts:1]: This was the error that was failing for the app to run. I got an error in the command saying can't compile the typescript. Will try this and see how it works.

Comment (AhhEagle) [reply backend/src/logger/api.logger.ts:1]: @shyakadev Well done. My approach was a bit different but I think this works as well.

**Review by AhhEagle — APPROVED** [review]
Well done @shyakadev . I think the changes are inline with what was discussed

File: backend/src/logger/api.logger.ts (hunk 1 above)
Comment (marksikaundi) [reply backend/src/logger/api.logger.ts:1]: Looks like Pine did us badly😂😂😂😂😂

Comment (shyakadev) [reply backend/src/logger/api.logger.ts:1]: @marksikaundi did you try

Comment (marksikaundi) [reply backend/src/logger/api.logger.ts:1]: @shyakadev yeah tried it

**Review by emmsdan — APPROVED** [review]

**Pull request merged**